### ISSUE TITLE
Debugging window will not be opened

### ISSUE THREAD

**Issue by neet** [Microsoft/vscode-generator-code#117]
Hello, When I'm trying to debug my extension with `Extension` of debug tab on activity bar this occurred. This Issue similar to #105, but the `watch` task seems working and compiled codes are output correctly.

<img width="1453" alt="2018-06-05 17 33 50" src="https://user-images.githubusercontent.com/19276905/40966822-80c5f462-68ec-11e8-9883-420d660ae465.png">

Now I'm avoiding this problem with deleting `preLaunchTask` of `Extension` in `launch.json` and run `npm run watch` manually and starting to debug.

I'm using vscode `1.23.1`, macOS `10.13.4`, node `9.11.1` yo `2.0.2` generator-code `1.1.32` and npm `6.1.0`. [This repository](https://github.com/neet/my-vscode-ext) is what was generated in my environment 

<details>
<summary>Logs when I generated the code</summary>

```
     _-----_     ╭──────────────────────────╮
    |       |    │   Welcome to the Visual  │
    |--(o)--|    │   Studio Code Extension  │
   `---------´   │        generator!        │
    ( _´U`_ )    ╰──────────────────────────╯
    /___A___\   /
     |  ~  |
   __'.___.'__
 ´   `  |° ´ Y `

? What type of extension do you want to create? New Extension (T
ypeScript)
? What's the name of your extension? my-vscode-ext
? What's the identifier of your extension? my-vscode-ext
? What's the description of your extension?
? What's your publisher name (more info: https://code.visualstud
io.com/docs/tools/vscecli#_publishing-extensions)? Neetshin
? Enable stricter TypeScript checking in 'tsconfig.json'? Yes
? Setup linting using 'tslint'? Yes
? Initialize a git repository? Yes
   create my-vscode-ext/.vscode/launch.json
   create my-vscode-ext/.vscode/settings.json
   create my-vscode-ext/.vscode/tasks.json
   create my-vscode-ext/src/test/extension.test.ts
   create my-vscode-ext/src/test/index.ts
   create my-vscode-ext/.vscodeignore
   create my-vscode-ext/.gitignore
   create my-vscode-ext/.gitattributes
   create my-vscode-ext/README.md
   create my-vscode-ext/CHANGELOG.md
   create my-vscode-ext/vsc-extension-quickstart.md
   create my-vscode-ext/tsconfig.json
   create my-vscode-ext/src/extension.ts
   create my-vscode-ext/package.json
   create my-vscode-ext/tslint.json
   create my-vscode-ext/.vscode/extensions.json


I'm all done. Running npm install for you to install the required dependencies. If this fails, try running the command yourself.



> my-vscode-ext@0.0.1 postinstall /Users/nucx/Desktop/my-vscode-ext
> node ./node_modules/vscode/bin/install

Detected VS Code engine version: ^1.23.0
Found minimal version that qualifies engine range: 1.23.0
Fetching vscode.d.ts from: https://raw.githubusercontent.com/Microsoft/vscode/7c7da59c2333a1306c41e6e7b68d7f0caa7b3d45/src/vs/vscode.d.ts
vscode.d.ts successfully installed!

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN my-vscode-ext@0.0.1 No repository field.
npm WARN my-vscode-ext@0.0.1 No license field.

added 265 packages from 488 contributors and audited 874 packages in 194.496s
found 0 vulnerabilities


Your extension my-vscode-ext has been created!

To start editing with Visual Studio Code, use the following commands:

     cd my-vscode-ext
     code .

Open vsc-extension-quickstart.md inside the new extension for further instructions
on how to modify, test and publish your extension.

For more information, also visit http://code.visualstudio.com and follow us @code.
```

</details>

**Comment by aromanovich** [comment]
I'm experiencing the same issue, vscode `1.23.1`, macOS `10.12.6`, node `9.4.0`, yo `2.0.2`, vscode-generator-code `1.1.32` and npm `5.6.0`.

I tried creating a dummy `sleep` task in my package.json:
```
"scripts": {
  ...
  "sleep": "sleep 20",
  ...
}
```
And specifying `"npm: sleep"` as a `preLaunchTask` in `.vscode/launch.json`.
As a result, the debugging VS Code instance won't start until `sleep 20` is done.

It makes me guess this issue isn't locale related one described in #105.

**Comment by dbaeumer** [comment]
This got fix for `1.24` and was caused by a change in the TypeScript version `2.9.x`. VS Code `1.23` works fine with TS < `2.9.x`
